src/app/tests/suites/certification/Test_TC_DISHM_2_1.yaml

```diff
@@ -13,28 +13,28 @@
 # limitations under the License.
 # Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
 
-name: 263.3.1. [TC-DISHM-2.1] ChangeToModeWithStatus Verification DUT as Server
+name: 193.2.1. [TC-DISHM-2.1] ChangeToMode Verification with DUT as Server
 
 PICS:
     - DISHM.S
 
 config:
     nodeId: 0x12344321
-    cluster: "Basic Information"
-    endpoint: 0
+    cluster: "Dishwasher Mode"
+    endpoint: 1
 
 tests:
-    - label:
-          "Step 1: Commission DUT to TH (can be skipped if done in a preceding
-          test)."
-      verification: |
-
-      disabled: true
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
 
-    - label: "Step 2: TH reads from the DUT the SupportedModes attribute."
-      PICS: DISHM.S.A0000
+    - label: "TH reads the SupportedModes attribute from DUT"
       verification: |
-          ./chip-tool dishwashermodeselect read supported-modes 1 1
+          ./chip-tool dishwashermode read supported-modes 1 1
 
           On TH(chip-tool) log, Verify SupportedModes attribute should contains two or more ModeOptionsStruct entries and
```

Comment:
Nothing in the spec requires the list to contain >= 2 entries.  This looks like a test plan issue; this should be guarded on some sort of PICS/PIXIT thing.  Please file that issue and reference it here.

Reply:
https://github.com/project-chip/connectedhomeip/issues/28323

Reply:
The spec was update via [this PR](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7371) to require >=2.